tests/cbs-xtpl-energy-conv/input.dat

```diff
@@ -34,7 +34,7 @@ compare_values(nucenergy_ref, h2o.nuclear_repulsion_energy(), 9, "Nuclear repuls
 
 scf_dz = energy('SCF/cc-pVDZ')
 compare_values(scf_dz_ref, scf_dz, 6, "[1] SCF/DZ Energy Check")  #TEST
-#clean()
+clean()
```

Comment:
yes, that's a manifestation of the below. there might have been an issue on it, too.

Pre-ddd, CBS wrapper issues a clean() btwn calcs. Plain string modelchem calcs like `energy('hf/cc-pvdz')` were getting caught and also being cleaned, meaning their behavior was slightly different than `set basis cc-pvdz \n energy('hf')`. With ddd, this is no longer happening, so for occasional string modelchem calcs, you may need to add a clean in the input. A few test cases now sport extra cleans.

Reply:
Okay, thanks. What "different behavior" is observed when the `clean()` isn't present in the input file? I don't normally `clean()` between calculations, and I expect most users don't either. If that is no longer the case, I think it would impact a lot of people.

Reply:
good point -- that bullet is vague. I've added an illustrative table to the PR frontmatter. fortunately the most common case will show up as PSIO error, a situation where one reflexively cleans. Only `mtd/bas` should need a change.